### ISSUE TITLE
Multiplayer not working

### ISSUE THREAD

**Issue by AlexanderJohnston** [KosmosisDire/TeammateRevive#38]
**Describe the bug**
When another player joins the lobby, their UI is messed up and they cannot select any characters or see what is going on. They cannot follow the host into the game.

**To Reproduce**
1. Install RoR2 and the new Void DLC
2. Install r2modman 3.1.27
3. Install Teammate Revival through the mod manager, the dependencies are included automatically.
4. Start the game and observe your log file:
```
    [Info   :   BepInEx] Loading [TeammateRevival 4.1.0]
    [Error  : Unity Log] TypeLoadException: Could not resolve type with token 0100001b (from typeref, class/assembly On.RoR2.Run, MMHOOK_Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null)
Stack trace:
TeammateRevive.MainTeammateRevival.Awake () (at <aeb998a1ba114dbfa5c12346a5ba4503>:0)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
FlashWindow:.cctor()
```
5. Observe the reported bug behavior in a multiplayer lobby by having another player with the same mods host a lobby for you to join.

**Expected behavior**
The ability to play in a listen server.

**Additional context**
Full log is attached including more detailed debugging statements
[LogOutput.log](https://github.com/KosmosisDire/TeammateRevive/files/8467096/LogOutput.log)
.

**Comment by AlexanderJohnston** [comment]
Log output from another client's perspective, same issue.

[R2LogOutput.log](https://github.com/KosmosisDire/TeammateRevive/files/8468542/R2LogOutput.log)

**Comment by AlexanderJohnston** [comment]
This may have been resolved. I was going to submit a PR but after looking through the code base I noticed there was nothing obviously broken.

I went ahead and deleted my r2modman profile and cleared out all data from the local appdata cache. It's possible there were some very old mod conflicts or leftover data.

When I have time later I'll have a friend connect to my listen server again and see if the problems are gone.
